en/backup-to-aws-s3-using-br.md

```diff
@@ -0,0 +1,525 @@
+
+4. Bind IAM to TiKV Pod:
+
+    - In the process of backup using BR, both TiKV Pod and BR Pod need to read and write S3 storage. Therefore, you need to add annotation to TiKV Pod to bind it with the IAM role:
+
+        {{< copyable "shell-regular" >}}
+
+        ```shell
+        kubectl edit tc demo1 -n test1
+        ```
+
+    - Find `spec.tikv.annotations`, append the `arn:aws:iam::123456789012:role/user` annotation, and then exit. After TiKV Pod is restarted, check whether the annotation is added to TiKV Pod.
+
+    > **Note:**
+    >
+    > `arn:aws:iam::123456789012:role/user` is the IAM role created in Step 4.
+
+#### Grant permissions by binding IAM with ServiceAccount
+
+1. Download [backup-rbac.yaml](https://github.com/pingcap/tidb-operator/blob/master/manifests/backup/backup-rbac.yaml), and execute the following command to create the role-based access control (RBAC) resources in the `test1` namespace:
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    kubectl apply -f backup-rbac.yaml -n test2
+    ```
+
+2. Create the `backup-demo1-tidb-secret` secret which stores the account and password needed to access the TiDB cluster:
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    kubectl create secret generic backup-demo1-tidb-secret --from-literal=password=<password> --namespace=test1
+    ```
+
+3. Enable the IAM role for the service account on the cluster:
+
+    - To enable the IAM role on your EKS cluster, refer to [Amazon EKS Documentation](https://docs.aws.amazon.com/eks/latest/userguide/enable-iam-roles-for-service-accounts.html)
+
+4. Create the IAM role:
+
+    - Create a IAM role and give the `AmazonS3FullAccess` permission to the role. Modify `Trust relationships` of the role. For details, refer to [Creating an IAM Role and Policy](https://docs.aws.amazon.com/eks/latest/userguide/create-service-account-iam-policy-and-role.html).
+
+5. Bind IAM to the ServiceAccount resources:
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    kubectl annotate sa tidb-backup-manager -n eks.amazonaws.com/role-arn=arn:aws:iam::123456789012:role/user --namespace=test1
+    ```
+
+6. Bind ServiceAccount to TiKV Pod:
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    kubectl edit tc demo1 -n test1
+    ```
+
+    Modify `spec.tikv.serviceAccount` to `tidb-backup-manager`. After TiKV Pod is restarted, check whether the `serviceAccountName` of TiKV Pod has changed.
+
+    > **Note:**
+    >
+    > `arn:aws:iam::123456789012:role/user` is the IAM role created in Step 4.
+
+### Process of ad-hoc full backup
+
+- If you grant permissions by accessKey and secretKey, create the `Backup` CR, and back up cluster data as described below:
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    kubectl apply -f backup-aws-s3.yaml
+    ```
+
+    The content of `backup-aws-s3.yaml` is as follows:
```

Comment:
@ran-huang Please check with the author whether the `kubectl apply -f backup-aws-s3.yaml` command should be executed after editing the yaml file. See [comment from Lilian](https://github.com/pingcap/docs-tidb-operator/pull/1#discussion_r394180644) in another PR.